cps/fb2.py

```diff
@@ -4,8 +4,10 @@
 from lxml import etree
 import os
 import uploader
-import StringIO
-
+try:
+    from io import StringIO
+except ImportError as e:
+    import StringIO
 
```

Comment:
ImportError available in Python2 and Python3  
ModuleNotFoundError is Python3 only

So need try to import Python3 module
and catch ImportError